### ISSUE TITLE
katib UI cannot collect training results and display,  still stop at first 3 trails, (kubeflow 1.5, katib  master)

### ISSUE THREAD

**Issue by sporterman** [kubeflow/katib#1836]
/kind bug

**What steps did you take and what happened:**
curl https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1beta1/hp-tuning/random.yaml --output random.yaml

download file,  download images, but i can not apply according to the example, because it cannot download datasets in container itself, so i enter the container and download datasets and commit a new imaegs kubeflowkatib/mxnet-mnist:v2, the file is as follow:

apiVersion: kubeflow.org/v1beta1
kind: Experiment
metadata:
  namespace: kubeflow-user-example-com
  name: random8
spec:
  objective:
    type: maximize
    goal: 0.94
    objectiveMetricName: Validation-accuracy
    additionalMetricNames:
      - Train-accuracy
  algorithm:
    algorithmName: random
  parallelTrialCount: 3
  maxTrialCount: 12
  maxFailedTrialCount: 3
  Metrics Collector Spec:
    Collector:
      Kind: StdOut
  parameters:
    - name: lr
      parameterType: double
      feasibleSpace:
        min: "0.01"
        max: "0.03"
    - name: num-layers
      parameterType: int
      feasibleSpace:
        min: "2"
        max: "5"
    - name: optimizer
      parameterType: categorical
      feasibleSpace:
        list:
          - sgd
          - adam
          - ftrl
  trialTemplate:
    primaryContainerName: training-container
    trialParameters:
      - name: learningRate
        description: Learning rate for the training model
        reference: lr
      - name: numberLayers
        description: Number of training model layers
        reference: num-layers
      - name: optimizer
        description: Training model optimizer (sdg, adam or ftrl)
        reference: optimizer
    trialSpec:
      apiVersion: batch/v1
      kind: Job
      spec:
        template:
          metadata:
            annotations:
              sidecar.istio.io/inject: "false"
          spec:
            containers:
              - name: training-container
                image: kubeflowkatib/mxnet-mnist:v2
                command:
                  - "python3"
                  - "/opt/mxnet-mnist/mnist.py"
                  - "--batch-size=64"
                  - "--lr=${trialParameters.learningRate}"
                  - "--num-layers=${trialParameters.numberLayers}"
                  - "--optimizer=${trialParameters.optimizer}"
            restartPolicy: Never

**What did you expect to happen:**
the katib ui display results as documentation, but i got this:
![katib-ui](https://user-images.githubusercontent.com/13059352/159220843-65fc3ba0-793c-475e-beee-329235601907.png)

**Anything else you would like to add:**
Validation-accuracy and Train-accuracy become 0 at katib ui when the training container just start working and train, 
![katib2](https://user-images.githubusercontent.com/13059352/159220881-579b9f29-1866-44d1-af56-f5bb0aa21d22.png)

**Environment:**

- Katib version (check the Katib controller image version):
- Kubernetes version: (`kubectl version`): 1.21
- OS (`uname -a`): Linux k8-master 5.4.0-100-generic #113~18.04.1-Ubuntu SMP Mon Feb 7 15:02:59 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

---

Impacted by this bug? Give it a 👍 We prioritize the issues with the most 👍

**Comment by charlescurt** [comment]
Same issue, any luck?

I would like to add the log files out of my master pod stuck in the running state.
<img width="888" alt="Screen Shot 2022-03-22 at 12 23 44 PM" src="https://user-images.githubusercontent.com/98784768/159565269-a93e0ecd-d89a-4616-a3d0-88cb6c01ca84.png">

**Comment by johnugeorge** [comment]
Please use the same issue to report all status #1837

**Comment by johnugeorge** [comment]
@sporterman Can you close this issue as issue is not valid anymore?

**Comment by johnugeorge** [comment]
Closing this issue as it works now
